packages/devtools_app/lib/src/performance/performance_model.dart

```diff
@@ -284,7 +292,15 @@ class OfflinePerformanceData extends PerformanceData {
     final CpuProfileData cpuProfileData =
         cpuProfileJson.isNotEmpty ? CpuProfileData.parse(cpuProfileJson) : null;
 
-    final String selectedFrameId = json[PerformanceData.selectedFrameIdKey];
+    final int selectedFrameId = json[PerformanceData.selectedFrameIdKey];
+
```

Comment:
nit: avoid dynamic where possible preferring Object so we get better static errors.